frontstage/models.py

```diff
@@ -65,9 +67,9 @@ def validate_phone_number(form, field):
     def validate_email_address(form, field):
         logger.debug('Checking if the email address contains a space')
         # this extends the email validator to check if there is whitespace in the email
-        if " " in field.data:
+        if ' ' in field.data:
             logger.debug('Space found in email address')
-            raise ValidationError("Your email should be of the form myname@email.com")
+            raise ValidationError('Your email should be of the form myname@email.com')
```

Comment:
Could the field be trimmed of any spaces at the beginning/end before being checking for spaces within the address?

Reply:
yes we can do it, probably in the scope of a different ticket though